devxicans/platform
# feat: configure theme for home page with font

## 🔥Configured theme for home page with font

### Description
I added custom styles from UiReact to use in our web page, and also import the font-family we're gonna be using.

### How to test?
You can run in the terminal: `pnpm run dev --filter=@1xdev/homepage` to see the web page runing with the syles and the font-family

### Screenshots
![image](https://github.com/user-attachments/assets/c279a267-dbfc-4a1f-a134-2b307c9e101a)
![image](https://github.com/user-attachments/assets/773af516-ee34-4ba8-82a3-f46c71aa6325)


#### Before

#### After
